keps/sig-ui/5008-kep-template/README.md

```diff
@@ -0,0 +1,949 @@
+desktop installations as there are different user profiles for each environment.
+For example, the plugin that allows managing plugins is not included in the
+in-cluster environment as the cluster admin generally determines which plugins
+to provide to the cluster users.
+
+### Existing Dashboard project
+
+The Kubernetes project already has a Dashboard project. However, we think that
+the Headlamp project has a number of advantages that make it a great solution
+to build on going forward. Some of these are:
+* Has a robust plugin system for customization and integrations
+* Supports both desktop and in-cluster mode
+* Is under very active development and has dedicated resources assigned to it
+  in addition to the active external contributors
+* Is built on a very common technology stack (Typescript, React, Material UI,
+  Vite, Go)
+* Has more advanced features (multi-cluster, graph view, powerful plugin
+  system, 3rd party plugins)
+* Addresses many more use cases (see below)
+
+### Goals
+
+<!--
+List the specific goals of the KEP. What is it trying to achieve? How will we
+know that this has succeeded?
+-->
+
+With this proposal we aim to provide the Kubernetes project with the following.
+* A tool to more easily onboard new and beginner-level users
+* A UI that serves as a base for vendors to build upon
+* A more modern UI developer environment
+* A more feature-rich UI
+* A desktop user experience
+* An integration point for other CNCF and external projects, especially those
+  without a UI focus
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+This proposal does not propose to force people who are using the current
+Dashboard to move to Headlamp.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+The proposal is to ship the Headlamp desktop experience as a new Kubernetes
+Desktop and the Headlamp in-cluster UI to eventually become the next generation
+of the Kubernetes Dashboard.
+
+Effectively this entails moving the Headlamp project under the Kubernetes
+GitHub organization and under the auspices of Kubernetes SIG UI. To be
+specific, we would propose that the Headlamp project be moved to a repo under
+the Kubernetes project; something to the effect of kubernetes/ui.
+The current repo kubernetes/dashboard would remain but include a note about the
+existence of the other repo. This would allow for users to compare the projects
+and make an informed decision about which works best for their use case. Over
+time, we can measure adoption trends and make decisions based on that data.
```

Comment:
As we already discussed, this will be a separate project under the same SIG. Neither one needs to mention the other projects directly in the project repo. It should be listed in: [https://github.com/kubernetes/community/tree/master/sig-ui](https://www.google.com/url?q=https://github.com/kubernetes/community/tree/master/sig-ui&sa=D&source=docs&ust=1736328081341321&usg=AOvVaw1ZkW4G1Fo-zShZ4hLIm0HX) and also the official kubernetes docs.